stories/card/fixed.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import { wInfo } from "../utils"
+
+import FixedCard from "../../src/components/card/fixed"
+
+storiesOf("Card", module).add(
+  "Fixed Card",
+  wInfo(`
+    Description
+    ~~~~
+    <FixedCard />
+    ~~~
+    `)(() => (
+    <div className="mx-30 mb-40">
+      <div className="text-2xl mb-20">Example</div>
+      <div className="w-3/12">
+        <FixedCard renderImage={() => <img src="/dummyImage.jpeg" />}>
+          <span className="pl-16 py-15">
```

Comment:
Is the padding something we could move to the component?
(If so we probably wanna use px instead of pl)

Reply:
umm....i would not do that since the component is just about the component and that span was just an example

Reply:
the padding is not part of the component.. but good eye 😄

Reply:
Yeah i though like maybe we every card should have the same base padding for its text content to be consisten.....

Reply:
I'd like having padding as part of a component here if that's feasible. Would help us with consistent look & feel.

Reply:
ok, then i will take a look

Reply:
even though i am not sure i can due to the fact that we have the image inside of the component and that will break the whole component

Reply:
so...if i add a wrapper around the children with the padding, and that children has a link, the clickable area won't be full width.

Reply:
We had a similar issue in `Button` and `Dropdown` we solved that with link wrapping - this could help: https://github.com/carforyou/carforyou-components-pkg/blob/master/src/lib/buttonHelper.tsx#L97

Reply:
padding are different: growing card has completly different paddings than the bank card. so for that case having padding in the card is not a feasible solution. and i would like to avoid having the fixed card with paddings and the growing card without paddings.

Reply:
I see you point on not adding the padding in a fixed card cuz there is now default padding in the growing card, you could also add it there. (Not that deep into the differences between the card types...)
If form a desing perspective a different padding is needed on the types of fixed cards you could either add options for the padding as in "small" | "large" or take the smaller of both paddins and use it as default and add the additional padding for the option needed

As a side note you could also go with a link function to avoid passing classed as in , this would also remove the need of rendering the link twice in for the listing cards.
```
const FixedCard = ({ renderImage, children, onClick, renderLink }) => {
  const renderComponent = () =>(
    <article className="fixedCard" onClick={onClick}>
      {renderImage()}
      {children}
    </article>
  )

  return renderLink ? renderLink(renderComponent) : renderComponent()
}
```

Reply:
i propose the following. i release with the current state, and then next sprint i will try to align with Enrique about the padding and then if they are the same for all the cards, i will make the changes or introduce small and big. WDYT?